scripts/debug-cli/src/main.rs

```diff
@@ -156,6 +156,7 @@ fn main() {
             use tokio::runtime::Runtime;
 
             let client = enclave_api::pruntime_client::new_pruntime_client(url);
+            let pubkey = pubkey.strip_prefix("0x").unwrap_or(&pubkey);
```

Comment:
Can use `decode_hex(pubkey)`

Reply:
Oh, just missed that there is a decode_hex. But I think its error message it not fit very well here.